presentation/src/main/java/com/architect/coders/mu8/characters/CharactersBindingAdapters.kt

```diff
@@ -0,0 +1,12 @@
+package com.architect.coders.mu8.characters
+
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.architect.codes.mu8.characters.Character
+
+@BindingAdapter("items")
+fun RecyclerView.setItems(characters: List<Character>?) {
```

Comment:
Didn't we said we were going to remove this type of methods?

Reply:
ok, delete file

Reply:
Esto vuela